packages/react-components/src/menu/src/MenuTrigger.tsx

```diff
@@ -107,18 +107,18 @@ export function InnerMenuTrigger({
         throw new Error("A menu trigger must have exactly 2 children.");
     }
 
-    const open = useCallback((event, focusTarget) => {
+    const open = useCallback((event: SyntheticEvent, focusTarget: string) => {
```

Comment:
For Menu, I wonder if a union type FocusTarget should be added?

I am not 100% sure because a focusTarget could be "first", "last" or any string key.

Une item, a union type should be added for FocusTarget with values being

Reply:
If we do add a union type, it could be used here